packages/api/package.json

```diff
@@ -109,7 +109,8 @@
     "whatwg-fetch": "^3.4.0",
     "winston": "^3.2.1",
     "wolfy87-eventemitter": "^5.2.9",
-    "yargs": "^17.3.1"
+    "yargs": "^17.3.1",
+    "@livepeer.studio/www": "0.15.0"
```

Comment:
Isn't this a cyclic dependency? Or is that fine?

Reply:
I thought that would be a problem too, but it's apparently fine in the context of a Yarn workspace. It helps that there's no actual circular import occurring — www imports api for certain schema definitions, and builds them into a bundle using the `next build` process; api then imports `server.js` from `www` which only serves out that compiled bundle.